supervision/annotators/core.py

```diff
@@ -830,9 +830,10 @@ class DotAnnotator(BaseAnnotator):
     def __init__(
         self,
         color: Union[Color, ColorPalette] = ColorPalette.DEFAULT,
-        radius: int = 4,
+        radius: int = 5,
```

Comment:
Let's leave the default value at `4`, even if it is hard to see.
People may already be using this in production systems.

Reply:
Cool, will keep it same.